docs/topics/azure-stack.md

```diff
@@ -109,6 +110,7 @@ Unless otherwise specified down below, standard [cluster definition](../../docs/
 | Name                            | Required | Description|
 | ------------------------------- | -------- | ---------- |
 | vmsize                          | yes      | Describes a valid [Azure Stack Hub VM size](https://docs.microsoft.com/azure-stack/user/azure-stack-vm-sizes). |
+| OSType                          | no       | Specifies the Operation System type of the agent pool. The default OSType value is `"Linux"`, and supported value is `"Windows"` for Windows OS. |
```

Comment:
`osType`

Reply:
I would just copy the text from `clusterdefinitions.md`

Specifies the agent pool's Operating System. Supported values are `Windows` and `Linux`. Defaults to `Linux`